airflow/example_dags/example_branch_python_dop_operator_3.py

```diff
@@ -46,17 +48,9 @@ def should_run(ds, **kwargs):
 
 
 cond = BranchPythonOperator(
-    task_id='condition',
-    provide_context=True,
-    python_callable=should_run,
-    dag=dag)
-
-oper_1 = DummyOperator(
-    task_id='oper_1',
-    dag=dag)
-oper_1.set_upstream(cond)
-
-oper_2 = DummyOperator(
-    task_id='oper_2',
-    dag=dag)
-oper_2.set_upstream(cond)
+    task_id='condition', provide_context=True, python_callable=should_run, dag=dag
+)
+
+oper_1 = DummyOperator(task_id='oper_1', dag=dag)
```

Comment:
Maybe change `oper_1` to `dummy_task_1`? This makes more sense when it shows up in the UI.

Reply:
I didn't refactor the task names but agree it makes more sense. Will fix.